tests/modules/UnitTest.Helpers.psm1

```diff
@@ -35,6 +35,9 @@ function Set-MockEnvironment {
     $ENV:apimPortalKeyVaultCertificateName = "test-apim-portal-com"
     $ENV:apimGatewayKeyVaultCertificateName = "test-apim-gateway-com"
     $ENV:apimManagementKeyVaultCertificateName = "test-apim-management-com"
+    $ENV:tenantPrimaryDomain = "tenant.onmicrosoft.com"
+    $ENV:apimAppRegistrationClientId = "fb0eac10-bda1-4410-9f8f-f4d381268d13"
```

Comment:
Is this a randomly generated GUID?

Reply:
I believe so, it's the same as the other GUID variables.